validate.go

```diff
@@ -21,6 +23,7 @@ import (
 var bundleValidateFlags = []cli.Flag{
 	cli.StringFlag{Name: "path", Value: ".", Usage: "path to a bundle"},
 	cli.BoolFlag{Name: "hooks", Usage: "Check specified hooks exist and are executable on the host."},
+	cli.BoolFlag{Name: "hook-specific", Usage: "Check host specified configs."},
```

Comment:
“hook-specific” → “host-specific”.  And I think the new flag should *replace* `--hooks`, not be an addition to it (because the reason we disable the “do you have that hook on *this* host?” check is [because it is host-specific][1]).  Is there a reason to keep the hook flag as an independent knob?

[1]: https://github.com/opencontainers/ocitools/pull/49#issuecomment-216585596